istioctl/cmd/root.go

```diff
@@ -143,7 +143,7 @@ debug and diagnose their Istio mesh.
 
 	kubeInjectCmd := kubeinject.InjectCommand(ctx)
 	hideInheritedFlags(kubeInjectCmd, cli.FlagNamespace)
-	rootCmd.AddCommand(kubeInjectCmd)
+	rootCmd.AddCommand(setNotSupportedCmd(kubeInjectCmd.Name()))
```

Comment:
Can we pass the whole `kubeInjectCmd` to the function and let it extract the name? This would make the code a bit cleaner.

Reply:
Done